### ISSUE TITLE
Remove warning if resolves in the develop section

### ISSUE THREAD

**Issue by vipulgupta2048** [balena-io/docs#2029]
Found in renamathon, 

https://github.com/balena-io/docs/blob/d2e7d4c691d3126418ff2cc1c45c42f32e282429/pages/learn/develop/runtime.md#L278-L279 I think we should remove or rephrase this to whatever is the fact right now.

**Comment by klutchell** [comment]
> At the time of writing there is an inconsistency in the behaviour of `/tmp` directory during reboot and application restart. With the current behaviour any thing in `/tmp` will persist over a reboot, but will **not** persist over an application restart.

AFAIK this is still the case due to the way we restart containers with the supervisor.

- `/tmp` in a container is ephemeral by default, both on balena and docker engines
- the container is maintained and restarted by the engine on a reboot, so no ephemeral layers are lost
- the container is destroyed and recreated by the supervisor on an application restart, so ephemeral layers are removed

We could test this again to be sure, but in cases where I wanted a true `/tmp` in my container I would add this to my compose file.

```yml
services:
  myapp:
    image: foo/bar
    tmpfs:
      - /tmp
```

So it could maybe be reworded to indicate that this applies to all ephemeral layers, and we could link to our supervisor app restart policy regarding data persistence and container recreation?

I certainly wouldn't call it an inconsistency. Writing to `/tmp` in a container on any platform will maintain that data between reboots until the container is removed and recreated. The only difference is our supervisor will do just that!

**Comment by klutchell** [comment]
We could also change the supervisor to always mount `/tmp` and `/var/tmp` as `tmpfs` in the background, but that would require a product brainstorm at the very least.

**Comment by vipulgupta2048** [comment]
No, I don't think we should the behavior by that much. It does make sense to me on how /tmp and /var/tmp behaves right now.
